core/include/scipp/core/variable.tcc

```diff
@@ -10,9 +10,14 @@
 #include "scipp/core/variable_view.h"
 #include "scipp/core/vector.h"
 #include "scipp/units/unit.h"
+#include <optional>
```

Comment:
Unused header?

Reply:
No this should be included.

Reply:
I am not sure what is used from it, is it `std::bad_variant_access`? If so it might be clearer to include `<variant>` instead?

Reply:
https://github.com/scipp/scipp/blob/6b5dd66e50e1ef3fef417be08106496c8f8e9bb3/core/include/scipp/core/variable.tcc#L53

Reply:
Ah okay, I thought it was related to the new additions. That makes snese